src/Serokell/Util/Base64.hs

```diff
@@ -92,3 +91,11 @@ instance FromJSON JsonByteStringDeprecated where
     parseJSON =
         parseJSON >=>
         either (fail . T.unpack) (pure . JsonByteStringDeprecated) . decodeUrl
+
+----------------------------------------------------------------------------
+-- Decoding helpers
+----------------------------------------------------------------------------
+
+mapLeft :: (a -> c) -> Either a b -> Either c b
```

Comment:
Yes, if we can drop cereal and binary deps here that'd be good. I don't think we have other packages using cereal and binary much (and if they still do, that's a good target for pruning too).